aldryn_newsblog/models.py

```diff
@@ -137,7 +149,7 @@ class Article(TranslatedAutoSlugifyMixin,
     tags = TaggableManager(blank=True)
 
     related = SortedManyToManyField('self', verbose_name=_('related articles'),
-                                    blank=True)
+                                    blank=True, symmetrical=ARTICLE_RELATED_SYMMETRICAL)
```

Comment:
It's about related-articles. With symmetrical=True if you say B relates to A (in A admin) you immediately get revered relation as well. Which is, to be honest, a bit surprising. Since I think no one thought about it before, simply assuming that there's only "B relates to A" connection, I'll change it to symmetrical=False and add a migration.

User will still be able to add the backward migration only from now on he/she 'll have to do it by hand.